src/driver/send_zc.rs

```diff
@@ -0,0 +1,82 @@
+use crate::driver::op::{self, Completable, Updateable};
+use crate::{
+    buf::IoBuf,
+    driver::{Op, SharedFd},
+    BufResult,
+};
+use std::{
+    io,
+    task::{Context, Poll},
+};
+
+pub(crate) struct SendZc<T> {
```

Comment:
That's a lot of work to get one new type of operation in that looks so clean in and of itself. So it goes.

Reply:
Well, it was a lot easier when it just allocated a smallvec 🙄